include/aspect/utilities.h

```diff
@@ -615,13 +615,15 @@ namespace aspect
         void
         declare_parameters (ParameterHandler  &prm,
                             const std::string &default_directory,
-                            const std::string &default_filename);
+                            const std::string &default_filename,
+                            const std::string &string = "Ascii data model");
```

Comment:
can we use a parameter name that is not ``string``? I find this really confusing.